meshroom/nodes/blender/RenderAnimatedCamera.py

```diff
@@ -1,47 +1,48 @@
-__version__ = "1.0"
+__version__ = "2.0"
 
 from meshroom.core import desc
 import os.path
 
 currentDir = os.path.dirname(os.path.abspath(__file__))
 
 class RenderAnimatedCamera(desc.CommandLineNode):
-    commandLine = '{blenderPathValue} -b --python {scriptPathValue} -- {allParams}'
+    commandLine = '{blenderCmdValue} -b --python {scriptValue} -- {allParams}'
 
-    category = 'Export'
+    category = 'Visualization'
     documentation = '''
         This node makes a rendering of the sfmData scene through an animated camera using the Blender rendering engine.
         It supports both Point Clouds (.abc) and Meshes (.obj).
     '''
 
     inputs = [
         desc.File(
-            name='blenderPath',
-            label='Blender Path',
-            description='''Path to blender executable''',
-            value=os.environ.get('BLENDER',"C:/Program Files/Blender Foundation/Blender 2.91/blender.exe"),
-            uid=[],
+            name='blenderCmd',
+            label='Blender Command',
+            description='Command to launch Blender',
+            value='blender',
+            uid=[0],
             group='',
         ),
         desc.File(
-            name='scriptPath',
+            name='script',
             label='Script Path',
-            description='''Path to the internal script for rendering in Blender''',
+            description='Path to the internal script for rendering in Blender',
             value=os.path.join(currentDir, 'scripts' ,'renderAnimatedCameraInBlender.py'),
-            uid=[],
+            uid=[0],
```

Comment:
We use an absolute path here. So I'm not sure if it should invalidate, as we may move the Meshroom install...